### DISCUSSION TITLE
Got the PCBs!

### DISCUSSION THREAD

**Discussion by caschy** [FundamentalFrequency/.github#22] [Show and tell]
So pumped to start this project
![20220624_131854](https://user-images.githubusercontent.com/18508692/175644613-3bfb40b6-8f5f-4baf-92b8-7e4783e3b354.jpg)
!

**Comment by reklm-e** [comment]
Do you need all 5 ?
If not i would really like to take one.

**Reply by caschy** [reply]
No problem! I only need 1.  I have someone else that wants another so that makes 3 left.

**Reply by reklm-e** [reply]
Do you want some Money or can u send it To me for free

**Reply by reklm-e** [reply]
and how can we make contact ?

**Reply by caschy** [reply]
I'm on the discord channel. msg me there.

**Reply by wawezz** [reply]
Hey i need one / 2. what's the price ?

**Reply by Kagesh** [reply]
Hello, are there any left? I would like to purchase one. Thanks!

**Comment by stonepreston** [comment]
awesome!

**Comment by seanarenouf** [comment]
@caschy where did you get these printed? trying to source some in the UK but cant seem to find anywhere that does it with direct upload of gerber (I do not have the technical knowledge to fill in a quotation form myself)

**Reply by caschy** [reply]
jlcpcb.com
